voice/bxml/verbs/playAudio.md

```diff
@@ -11,8 +11,14 @@ always cache for a shorter amount of time or not cache at all. If no `Cache-Cont
 header is set on the response, media will not be cached.
 
 The audio format is determined by the HTTP `Content-Type` header in the response. Our system supports:
-- `audio/wav` and `audio/x-wav` for .wav files encoded as PCM or G711
-- `audio/mpeg`, `audio/mpeg3` and `audio/mp3` for .mp3 files
+- `audio/wav` and `audio/x-wav` for `.wav` files
+  - Both `G711 μ-law` and `G711 A-law` are supported within the `pcm_s16le` container (signed, 16-bit, little-endian, PCM-encoded `.wav` file)
+- `audio/mpeg`, `audio/mpeg3`, and `audio/mp3` for `.mp3` files
+  - MPEG-1 layer 3 (`8`, `11.025`, and `12` kHz), MPEG-2 layer 3 (`16`, `22.05`, and `24` kHz), and MPEG-2.5 layer 3 (`32`, `44.1`, and `48` kHz) are all supported
```

Comment:
We should put the ID3 tag max here as well

Reply:
The limit is on each `ID3v2` tag, right? Not the total size of all of them? And ID3v1 doesn't apply either, right?

Reply:
An ID3v2 tag is the whole thing (the small pieces are called frames), so the limit is on the whole thing. And doesn't apply to ID3v1 as that is only 128 bytes